db/document_test.go

```diff
@@ -222,3 +223,40 @@ func TestParseDocumentCas(t *testing.T) {
 
 	goassert.Equals(t, casInt, uint64(1492749160563736576))
 }
+
+func TestPruneRevisionsWithDisconnected(t *testing.T) {
```

Comment:
Rename and comment for clarity, but also should this test be in `db/revtree_test.go` with all the other revtree pruning tests?

```suggestion
// TestPruneRevisionsWithDisconnectedTombstones ensures that disconnected tombstoned branches are correctly pruned away. Reproduces CBG-1076.
func TestPruneRevisionsWithDisconnectedTombstones(t *testing.T) {
```